httpserver-rs/src/settings.rs

```diff
@@ -195,6 +199,25 @@ pub fn load_settings(values: &HashMap<String, String>) -> Result<ServiceSettings
         settings.merge(ServiceSettings::from_json(str.as_bytes())?);
     }
 
+    // accept address as value parameter
+    if let Some(addr) = values.get("address") {
```

Comment:
By convention previously we've had all of the values be uppercase, if we want to stick with that I'd suggest changing this to `ADDRESS`.

I'm not a fan of changing everything in past providers, but I'd be OK with making values case insensitive. It would be up to the provider developer to do this though.